presto-hive/src/main/java/com/facebook/presto/hive/security/SqlStandardAccessControl.java

```diff
@@ -411,6 +419,10 @@ private boolean checkTablePermission(ConnectorTransactionHandle transaction, Ide
 
     private boolean hasGrantOptionForPrivilege(ConnectorTransactionHandle transaction, Identity identity, Privilege privilege, SchemaTableName tableName)
     {
```

Comment:
Is it true that the `admin` role has GRANT OPTION implicitly assigned to it? Do the Hive specs say so?